site/src/pages/ManagementSettingsPage/CustomRolesPage/CustomRolesPage.tsx

```diff
@@ -32,7 +30,10 @@ export const CustomRolesPage: FC = () => {
 	);
 	const [roleToDelete, setRoleToDelete] = useState<Role>();
 	const organizationRolesQuery = useQuery(organizationRoles(organizationName));
-	const filteredRoleData = organizationRolesQuery.data?.filter(
+	const builtInRoles = organizationRolesQuery.data?.filter(
+		(role) => role.built_in === true,
+	);
+	const customRoles = organizationRolesQuery.data?.filter(
 		(role) => role.built_in === false,
 	);
```

Comment:
```suggestion
	const builtInRoles = organizationRolesQuery.data?.filter(
		(role) => role.built_in,
	);
	const customRoles = organizationRolesQuery.data?.filter(
		(role) => !role.built_in,
	);
```

they're already booleans, feels unnecessary to be so explicit 🤷‍♀️